<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerHardwareDeviceWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerHardwareDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-3</y>
        <width>757</width>
        <height>207</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="1">
           <widget class="QWidget" name="authTypeButtonsWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="hostNameLabel">
            <property name="text">
             <string>&amp;Host name:</string>
            </property>
            <property name="buddy">
             <cstring>hostLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="hostLineEdit">
              <property name="placeholderText">
               <string>IP or host name of the device</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="userNameLabel">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>userLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="userLineEdit"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="sshPortLabel">
            <property name="text">
             <string>&amp;SSH port:</string>
            </property>
            <property name="buddy">
             <cstring>sshPortSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QSpinBox" name="sshPortSpinBox">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>22</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="connectionTimeoutLabel">
            <property name="text">
             <string>Timeout:</string>
            </property>
            <property name="buddy">
             <cstring>timeoutSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QSpinBox" name="timeoutSpinBox">
              <property name="suffix">
               <string>s</string>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="freePortsLabel">
            <property name="text">
             <string>Free ports:</string>
            </property>
            <property name="buddy">
             <cstring>portsLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="portsLineEdit">
              <property name="toolTip">
               <string>You can enter lists and ranges like this: 1024,1026-1028,1030</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="portsWarningLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="authTypeLabel">
            <property name="text">
             <string>Authentication:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="autheticationTypeLabelEdit">
              <property name="text">
               <string>none</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="keyLabel">
            <property name="text">
             <string>Private SSH Key:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="privateKeyLabelEdit">
              <property name="text">
               <string>none</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
